src/checkbox/checkbox.component.ts

```diff
@@ -60,12 +80,25 @@ export class IgxCheckboxComponent implements ControlValueAccessor {
         this.checked = !!this._value;
     }
 
+    public get labelClass(): string {
+        switch (this.labelPosition) {
+            case LabelPosition.BEFORE:
+            case "before":
```

Comment:
If the `LabelPosition` is a string enumeration then this check is kind of redundant.